go/private/rules/library.bzl

```diff
@@ -44,7 +44,10 @@ def _go_library_impl(ctx):
         source,
         archive,
         DefaultInfo(
-            files = depset([archive.data.file]),
+            files = depset([
+                archive.data.file,
```

Comment:
How about only advertising the `export_file` here? Having multiple file outputs always makes a rule more tedious to use.

Reply:
Sounds good, as long as we're not depending on this file output.